### PR TITLE
Fix Documentation: PR11143

### PR DESCRIPTION
**Pull request by ghost**
Closes https://github.com/RocketChat/Rocket.Chat/issues/11143

### CODE REVIEW

File: developer-guides/rest-api/authentication/login/README.md [hunk 1]
```diff
@@ -18,69 +18,53 @@
...
-     -d "username=myusername&password=mypassword"
-```
-
-```bash
-curl http://localhost:3000/api/v1/login \
-     -d "user=myusername&password=mypassword"
-```
-
-```bash
-curl http://localhost:3000/api/v1/login \
```
Comment (MarcosSpessatto) [inline developer-guides/rest-api/authentication/login/README.md:33]: Why did you delete the calls with username and password?

File: developer-guides/rest-api/authentication/login/README.md [hunk 2]
```diff
@@ -18,69 +18,53 @@
...
-```
-
-```bash
-curl -H "Content-type:application/json" \
-      http://localhost:3000/api/v1/login \
-      -d '{ "user": "myusername", "password": "mypassword" }'
-```
-
-```bash
-curl -H "Content-type:application/json" \
```
Comment (MarcosSpessatto) [inline developer-guides/rest-api/authentication/login/README.md:52]: Why did you delete the calls with username and password?

**Review by MarcosSpessatto — CHANGES_REQUESTED** [review]
I think you could leave this file with the login descriptions with `username` and `password` as it was, and make a new section for `ldap`, as it already exists in [facebook](https://rocket.chat/docs/developer-guides/rest-api/authentication/facebook/) for example. What do you think @samilaine ?

**Comment by MartinSchoeler** [comment]
@samilaine Could you please address the reviews? Thanks!

**Comment by MartinSchoeler** [comment]
Closing this PR due lack of response from the author.
